### PR TITLE
Add tutorial for audit policy

### LINKED ISSUE #4488: How to enable auditing?
**The exact command to reproduce the issue**:
```
minikube start \
--vm-driver hyperkit \
--extra-config=apiserver.authorization-mode=RBAC \
--extra-config=apiserver.Audit.LogOptions.Path=/var/log/apiserver/audit.log \
--extra-config=apiserver.Audit.PolicyFile=/etc/kubernetes/addons/audit-policy.yaml
```
I tried several extra-config variants:
- apiserver.Audit.LogOptions.Path, apiserver.Audit.PolicyFile
- apiserver.audit-logoptions-path, apiserver.audit-policyfile
- apiserver.audit-log-options-path, apiserver.audit-policy-file
- apiserver.audit-policy-path, apiserver.audit-log-dir, apiserver.audit-log-max-age

together with different values:
- log options path / log path / path
  - /var/log/kube-apiserver-audit.log
  - /etc/kubernetes/logs/apiserver-audit.log
- log dir
  - /var/log/kubernetes/
- policy path
  - /etc/kubernetes/addons/audit-policy.yaml

I tried also:
- `--feature-gates=Auditing=true`
- `--feature-gates=AdvancedAuditing=true`

It does not matter how many CPUs or memory I give, minikube does not start in any case :(

**The full output of the command that failed**:
Please see all attached files

**The output of the `minikube logs` command**:
No output because minikube does not start.

**The operating system version**:
MacBook Pro - macOS Mojave 10.14.5
Minikube v1.1.1

I already had a look at the issue #1609, but it doesn't helped :(

I attached all attempts I did.
If someone has ideas or can help it would be great!
Thanks guys!!

[attempt-1.log](https://github.com/kubernetes/minikube/files/3291069/attempt-1.log)
[attempt-2.log](https://github.com/kubernetes/minikube/files/3291070/attempt-2.log)
[attempt-3.log](https://github.com/kubernetes/minikube/files/3291071/attempt-3.log)
[attempt-4.log](https://github.com/kubernetes/minikube/files/3291072/attempt-4.log)
[attempt-5.log](https://github.com/kubernetes/minikube/files/3291073/attempt-5.log)
[attempt-6.log](https://github.com/kubernetes/minikube/files/3291074/attempt-6.log)
[attempt-7.log](https://github.com/kubernetes/minikube/files/3291075/attempt-7.log)
[attempt-8.log](https://github.com/kubernetes/minikube/files/3291077/attempt-8.log)
[attempt-9.log](https://github.com/kubernetes/minikube/files/3291078/attempt-9.log)
[attempt-10.log](https://github.com/kubernetes/minikube/files/3291079/attempt-10.log)

### PR DESCRIPTION
**Pull request by JanAhrens**
This tutorial was assembled from information provided in the issue and is supposed to fix #4488

### CODE REVIEW

**Comment by medyagh** [comment]
Thanks for adding this tutorial! This was one of the frequently asked questions .

**Comment by JanAhrens** [comment]
@medyagh It seems that this PR is still missing the `lgtm` and `approved` label. Can you let me know what needs to be changed to get this merged?

**Comment by medyagh** [comment]
@JanAhrens yes, please change the path for the yaml file to somewhere other than cert folder currently it is in `/etc/ssl/certs/audit-policy.yaml`
I rather we don't mix the yaml and certs

File: site/content/en/docs/Tutorials/audit-policy.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+to the minikube API server on startup.
+
+## Tutorial
+
+```shell
+minikube stop
+
+mkdir -p ~/.minikube/files/etc/ssl/certs
+
+cat <<EOF > ~/.minikube/files/etc/ssl/certs/audit-policy.yaml
```
[the remark this answers is not in the document]
Comment (bygui86) [reply site/content/en/docs/Tutorials/audit-policy.md]: Honestly I tried in other folders:
`~/.minikube/files/audit-policy.yaml`
`~/.minikube/files/auditing/audit-policy.yaml`
`~/.minikube/files/audit/audit-policy.yaml`
it doesn't work...
Maybe `~/.minikube/files/etc/audit-policy.yaml`?

[the remark this answers is not in the document]
Comment (JanAhrens) [reply site/content/en/docs/Tutorials/audit-policy.md]: > Any reason to put th yaml file in the cert folder ?

It's one of the folders that get mounted from the minikube host:
```
$ kubectl get pod kube-apiserver-minikube -n kube-system -o json | jq '.spec.volumes[].hostPath.path'
"/etc/ssl/certs"
"/var/lib/minikube/certs"
"/usr/share/ca-certificates"
```

I guess there's a way to specify more volumes to be mounted by the container, but honestly I don't know how to do that. It seems that kubeadm takes care of this.

I personally think that abusing`/etc/ssl/certs` for this task isn't very nice, but it's a way to enable audit logging without changing much within minikube. What do you think @medyagh?

[the remark this answers is not in the document]
Comment (medyagh) [reply site/content/en/docs/Tutorials/audit-policy.md]: I see this is a kube-apiserver config, and currently thats the only folder we mount. given that not everyone might need to mount a hostpath to api server I am okay with using this folder. I would suggest a comment on the tutorial that why we are doing this.

**Comment by medyagh** [comment]
@JanAhrens 

I suggest adding a comment in the footnotes explaining why chose cert folder for the manifest( the reason is that is one of the default hostPath folders mounted by minikube)

to make it more clear

File: site/content/en/docs/Tutorials/audit-policy.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bygui86) [reply site/content/en/docs/Tutorials/audit-policy.md]: I've seen in many tutorials about minikube that "it's possible to mount files in the folder '~/.minikube/files' and they are going to be available in minikube"... So this is not working right?

**Comment by JanAhrens** [comment]
Thanks for your review @medyagh. I tried to make it clearer why the `/etc/ssl/certs`directory gets used in this tutorial. Can you give it another read?

/assign @medyagh

**Pull request merged**